extensions/src/main/java/org/pentaho/platform/plugin/services/email/EmailService.java

```diff
@@ -73,23 +96,44 @@ public class EmailService implements IEmailService {
    */
   private File emailConfigFile;
 
+  private static final String REFRESH_TOKEN = "refresh_token";
+  private static final String AUTHORIZATION_CODE = "authorization_code";
+
+  private static final String CLIENT_CREDENTIALS = "client_credentials";
+  private static final String GRANT_TYPE = "grant_type";
+  private static final String SCOPE = "scope";
+  private static final String CLIENT_ID = "client_id";
+  private static final String CLIENT_SECRET = "client_secret";
+  private static final String CODE = "code";
+  private static final String XOAUTH = "XOAUTH2";
+  private static final String REDIRECT_URI = "redirect_uri";
+  private static final String TEST_EMAIL_FAIL = "EmailTester.FAIL";
+  private static final String TEST_EMAIL_SUCCESS = "EmailTester.SUCESS";
```

Comment:
great job creating java const `TEST_EMAIL_FAIL` and  `TEST_EMAIL_SUCCESS`. The fact that's it's misspelled "_SUCESS_" is why we create constants.

We should use the java consts in other places:

- https://github.com/NJtwentyone/pentaho-platform/blob/master/user-console/src/main/java/org/pentaho/mantle/client/admin/EmailTester.java#L74
- https://github.com/NJtwentyone/pentaho-platform/blob/master/user-console/src/main/java/org/pentaho/mantle/client/admin/EmailTester.java#L76
_ and any other place